contracts/HeapOrdering.sol

```diff
@@ -141,18 +141,22 @@ library HeapOrdering {
         Account memory accountToShift = getAccount(_heap, _rank);
         uint256 valueToShift = accountToShift.value;
         Account memory childToSwap;
+        Account memory rightChild;
```

Comment:
did you try internalizing those avariables inside the loop? i know it sounds a bit weird but sometimes it saves some gas

Reply:
Sorry but I won't do it without a rationale behind it. Otherwise we could spend a lot of time not understanding what we are doing

Reply:
the rational is that if we don't enter the loop we don't instantiate something useless in memory

Reply:
but ok with your choice